prime-router/cleanslate.sh

```diff
@@ -67,6 +70,10 @@ Examples:
   # Use this if you like chasing red herrings in debug-land
   $ ${0} --keep-images
 
+  # Use this if you like to change ownership of some directories that are
+  # potentially shared with container instances
+  $ ${0} --retake-ownership 
```

Comment:
A good example here would be `${0} --keep-all --take-ownership` to _just_ take ownership

Reply:
Change made